### ISSUE TITLE
[BITV] 9.1.3.1b/3.2 - The breadcrumb navigation represents a list and should best be implemented as such. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36918]
See https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/ for more details. 

- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1d5eadb58d-v3-n2

</details>

- [ ] 9\.1.3.1b/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1d5eadb58d-v10-n2

**Comment by JuliaKirschenheuter** [comment]
Hi @michaelnissenbaum 

is such solution ok in your eyes? 

1. Is text of `aria-label` for navigation ok?

|  Before   | After                                                        |
|-------------|---------------------------------------------------------------|
| ![Screenshot from 2023-03-28 17-24-34](https://user-images.githubusercontent.com/6078378/228287765-ffb7adf1-f792-4602-b91c-bdfee47b51e5.png) | ![Screenshot from 2023-03-29 13-38-08](https://user-images.githubusercontent.com/6078378/228523968-68668ee5-326f-454a-b4e9-7e1d7ca98fad.png)

2. Is it ok to have an action button inside of `<div>` which is inside of `<nav>`? Bacause it is actually not a part of a list.

After:

![Screenshot from 2023-03-29 13-33-40](https://user-images.githubusercontent.com/6078378/228523743-2af464a1-91e5-4df6-be21-16294e300111.png)

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter 

1. The aria-label attribute was used correctly in your example. See also https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/. However, what I would recommend is removing the word "navigation" since the role is automatically announced by the screen reader because a nav element was used.
2. In my opinion, the action button should not be part of the breadcrumb navigation. This could confuse screen reader users. Please follow the official WAI-ARIA pattern.

**Comment by JuliaKirschenheuter** [comment]
Thank you @michaelnissenbaum!
